syntax = "proto3";

import "rescode.proto";

package ocean_mprpc;

option cc_generic_services = true;

message GetFriendListRequest{
    uint32 userid=1;
}

message GetFriendListReponse{
    ocean_mprpc.ResultCode res=1;
    repeated bytes friends=2;
}

service FriendServiceRpc{
    rpc GetFriendList(GetFriendListRequest) returns(GetFriendListReponse);
}